hack/generated/pkg/genruntime/base_types.go

```diff
@@ -57,6 +57,12 @@ type KubernetesResource interface {
 
 	// TODO: I think we need this
 	// SetStatus(status interface{})
+
+	// GetSpec returns the specification of the resource
+	GetSpec() ConvertibleSpec
+
+	// GetStatus returns the current status of the resource
```

Comment:
minor: Same comment as above, one of these says "current X" and one says "X".

I think just `GetStatus returns the status of the resource` is good enough?